src/NUnitConsole/nunit3-console.tests/nunit3-console.tests.csproj

```diff
@@ -7,7 +7,7 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="NSubstitute" Version="2.0.3" />
-    <PackageReference Include="NUnit" Version="3.11.0" />
+    <PackageReference Include="NUnit" Version="3.13.0-dev-06899" />
```

Comment:
I guess that this is just until the 3.14 release?

Reply:
Yep - although hopefully with MyGet it won't matter too much to any external users, so will probably just upgrade when we get round to it. We need the latest master for the .NET 5.0 fix for PlatformAttribute.